scripts/keras_benchmarks/models/lstm_text_generation_benchmark.py

```diff
@@ -0,0 +1,104 @@
+'''
+Original Model from keras/examples
+
+Example script to generate text from Nietzsche's writings.
```

Comment:
I think this script wouldn't be a good fit for a benchmark, since we don't need to train on real data (randomly generated numpy data will be fine). Most of the script can be edited away.